tests/Api/Admin/TaxonImagesTest.php

```diff
@@ -24,61 +24,6 @@ final class TaxonImagesTest extends JsonApiTestCase
 {
     use AdminUserLoginTrait;
 
-    /** @test */
-    public function it_gets_taxon_images(): void
-    {
-        $this->loadFixturesFromFiles(['authentication/api_administrator.yaml', 'taxon_image.yaml']);
-        $header = array_merge($this->logInAdminUser('api@example.com'), self::CONTENT_TYPE_HEADER);
-
-        $this->client->request(method: 'GET', uri: '/api/v2/admin/taxon-images', server: $header);
-
-        $this->assertResponse(
-            $this->client->getResponse(),
-            'admin/taxon_image/get_taxon_images_response',
-            Response::HTTP_OK,
-        );
-    }
-
-    /** @test */
-    public function it_gets_taxon_images_with_an_image_filter(): void
-    {
-        $this->loadFixturesFromFiles(['authentication/api_administrator.yaml', 'taxon_image.yaml']);
-        $header = array_merge($this->logInAdminUser('api@example.com'), self::CONTENT_TYPE_HEADER);
-
-        $this->client->request(
-            method: 'GET',
-            uri: '/api/v2/admin/taxon-images',
-            parameters: [ImageNormalizer::FILTER_QUERY_PARAMETER => 'sylius_small'],
-            server: $header,
-        );
-
-        $this->assertResponse(
-            $this->client->getResponse(),
-            'admin/taxon_image/get_taxon_images_with_image_filter_response',
-            Response::HTTP_OK,
-        );
-    }
-
-    /** @test */
-    public function it_prevents_getting_taxon_images_with_an_invalid_image_filter(): void
-    {
-        $this->loadFixturesFromFiles(['authentication/api_administrator.yaml', 'taxon_image.yaml']);
-        $header = array_merge($this->logInAdminUser('api@example.com'), self::CONTENT_TYPE_HEADER);
-
-        $this->client->request(
-            method: 'GET',
-            uri: '/api/v2/admin/taxon-images',
-            parameters: [ImageNormalizer::FILTER_QUERY_PARAMETER => 'invalid'],
-            server: $header,
-        );
-
-        $this->assertResponse(
-            $this->client->getResponse(),
-            'common/image/invalid_filter',
-            Response::HTTP_BAD_REQUEST,
-        );
-    }
-
```

Comment:
Why are you removing the tests for getting a collection?

Reply:
I'm resolving as the appropriate tests for getting a collection for given taxon already exist 🖖🏻